tests/Functional/Fixtures/web/symfony.php

```diff
@@ -21,4 +21,4 @@
 $request = Request::createFromGlobals();
 $response = $kernel->handle($request);
 $response->send();
-$kernel->terminate($request, $response);
+//$kernel->terminate($request, $response);
```

Comment:
hm, so indeed at the time we try to terminate the kernel, the dispatcher inside the kernel is set to null. i have no idea why or even how this can happen.

it would sound like a general symfony + hhvm problem, but google does not bring up the problem.

```
Fatal error: Uncaught exception 'BadMethodCallException' with message 'Call to a member function dispatch() on a non-object (NULL)' in /vagrant_data/FOSHttpCache/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php:86
Stack trace:
#0 /vagrant_data/FOSHttpCache/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpCache/HttpCache.php(245): Symfony\\Component\\HttpKernel\\HttpKernel->terminate()
#1 /vagrant_data/FOSHttpCache/tests/Functional/Fixtures/web/symfony.php(24): Symfony\\Component\\HttpKernel\\HttpCache\\HttpCache->terminate()
#2 {main}
```